packages/fluent/scss/list/_variables.scss

```diff
@@ -184,13 +184,13 @@ $kendo-list-sizes: (
 
 /// Background color of the list component.
 /// @group list
-$kendo-list-bg: var( --kendo-component-bg, #{$kendo-component-bg} ) !default;
+$kendo-list-bg: var( --kendo-component-bg, transparent ) !default;
 /// Text color of the list component.
 /// @group list
-$kendo-list-text: var( --kendo-component-text, #{$kendo-component-text} ) !default;
+$kendo-list-text: var( --kendo-component-text, transparent ) !default;
```

Comment:
the text and border variables should not fallback to transparent color (as they might become invisible if the variable is not defined). It's okay for the background color, but the text and border should be either inherit or initial.